src/key_info_managers/on_disk_manager/mod.rs

```diff
@@ -187,12 +187,14 @@ impl OnDiskKeyInfoManager {
         for app_name_dir_path in list_dirs(&mappings_dir_path)?.iter() {
             for provider_dir_path in list_dirs(&app_name_dir_path)?.iter() {
                 for key_name_file_path in list_files(&provider_dir_path)?.iter() {
-                    info!("Found mapping file: {:?}.", key_name_file_path);
+                    if crate::utils::GlobalConfig::log_error_details() {
+                        info!("Found mapping file: {:?}.", key_name_file_path);
+                    }
```

Comment:
Even if `log_error_details` is not set, it would be nice to know that *a* mapping file was found. Maybe this is the case for a `format_info` macro?

And `format_warning` for completeness 🤡

Reply:
The thing is, for errors `format_error` makes sense because we want to potentially display the error, but for warning and info it tends to be various bits of data that we might or might not want to log

Reply:
Having 100 `Found mapping file` seems weird, but I can add a `Found {} mapping files` and the count just before returning from this method?

Reply:
The reason why in some cases I decided to not print anything is because it seems like it's not adding much value, but I'm open to reconsidering them!

Reply:
When testing persistence, it is nice to see that keys were recognised and added, a count might help then!

Reply:
Added here: https://github.com/parallaxsecond/parsec/pull/183/files#diff-1b1e7f5d68119cb3267c219b6be4ce9eR225

Reply:
Nice, thanks!